common/app/Panes/Panes.jsx

```diff
@@ -44,6 +44,8 @@ const propTypes = {
   render: PropTypes.func.isRequired
 };
 
+const quoteObj = require('./quotes').fetchRandomQuote();
```

Comment:
I would still like this to use `import` syntax so we have consistency in the file.

```js
import { fetchRandomQuote } from './quotes';

const quoteObj = fetchRandomQuote();
```

Reply:
Ah, sorry about that! I see what you mean now. I can make that change.